tests/e2e/src/pages/contacts.page.ts

```diff
@@ -1,8 +1,6 @@
 import { BasePage } from "./base.page";
-import { Helper } from "../helpers/helper";
 
 import type { ICustomWorld } from "../support/custom-world";
-let selector: string;
```

Comment:
Removed same as issue above - after moving method from contacts.ts - selector variable never used an lint asked to remove it.